packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -510,7 +506,7 @@ abstract class FlutterCommand extends Command<void> {
 
     String extraFrontEndOptions =
         argParser.options.containsKey(FlutterOptions.kExtraFrontEndOptions)
-            ? stringArg(FlutterOptions.kExtraFrontEndOptions)
+            ? stringsArg(FlutterOptions.kExtraFrontEndOptions)?.join(',')
```

Comment:
It looks like these arguments are not tested or specified correctly, still working on what they should actually be. Tempted to remove for now